extide/gradle/src/org/netbeans/modules/gradle/spi/newproject/TemplateOperation.java

```diff
@@ -488,7 +488,7 @@ public Set<FileObject> execute() {
                     args.add("--gradle-version"); //NOI18N
                     args.add(version);
                 }
-                pconn.newBuild().withArguments("--offline").forTasks(args.toArray(new String[0])).run(); //NOI18N
+                pconn.newBuild().forTasks(args.toArray(new String[0])).run(); //NOI18N
```

Comment:
Probably it would be better to honor [`GradleSettins.isOffline()`](https://github.com/apache/netbeans/blob/681aa7656757ed9ddc7c2f19af8360f85dcfdf66/extide/gradle/src/org/netbeans/modules/gradle/spi/GradleSettings.java#L234)

Reply:
@lkishalmi OK, I've made both the init and wrapper tasks only add `--offline` if the global option is set. Please re-review.

Of course, the Tooling API isn't offline itself, and I'm not sure if it has a configuration option to fail rather than download things?